backend/tests/gsr_booking/test_gsr_views.py

```diff
@@ -82,8 +82,8 @@ def test_recent(self):
         response = self.client.get(reverse("recent-gsrs"))
         res_json = json.loads(response.content)
         self.assertEquals(2, len(res_json))
-        self.assertEquals(6, len(res_json[0]))
-        self.assertEquals(6, len(res_json[1]))
+        self.assertEquals(7, len(res_json[0]))
+        self.assertEquals(7, len(res_json[1]))
```

Comment:
i'm imagining that the extra field is the `in_use`? any reason it still shows up in the route despite all of our queries using `.object` to query? if this is the case, I think we should remove this field from showing up

Reply:
oh the serializer was still serializing all the fields. fixed now

Reply:
bet LGTM